tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs

```diff
@@ -59,7 +59,7 @@ public async void Summary_UserHasTwoWorkspaces_ReturnsSummaryAggregatedByWorkspa
             string summary = await reportSummaryService.Summary(Channels.Telegram, userProfile, dateRange);
 
             // Assert
-            summary.Should().BeEquivalentTo($"You worked **2.72d** in 01 January 2020 - 01 March 2020" +
+            summary.Should().BeEquivalentTo($"Between 01 January 2020 - 01 March 2020, you worked **2.72d** (21) hours" +
```

Comment:
You should write the assertion with the new message format, like 2.72 days (21 hours).